src/xbot/common/math/PIDPropertyManager.java

```diff
@@ -129,4 +146,12 @@ public boolean getEnableDerivativeThreshold() {
     public void setEnableDerivativeThreshold(boolean isEnabled) {
         propEnableDerivativeThreshold.set(isEnabled);
     }
+    
+    public boolean getEnableTimeThreshold() {
+        return propEnableTimeThreshold.get();
```

Comment:
The enable/disable is more to support if you want to temporarily disable threshold checking, but want to keep your configured threshold.